test/suite_test.go

```diff
@@ -32,22 +32,24 @@ import (
 )
 
 const (
-	kindVersion = "v0.8.1"
-	kindImage   = "kindest/node:v1.18.2"
-	testImage   = "securityoperators/seccomp-operator:latest"
+	kindVersion    = "v0.8.1"
+	kubectlVersion = "v1.18.5"
+	kindImage      = "kindest/node:v1.18.2"
+	testImage      = "securityoperators/seccomp-operator:latest"
```

Comment:
more of a question: would it be worth having the current branch's image being used?

Reply:
It will be used, we build it locally and kind loads the image into the node :) This means every code change should be part of the e2e test.